c7n/resources/glue.py

```diff
@@ -186,6 +187,53 @@ class resource_type(TypeInfo):
     augment = universal_augment
 
 
+@GlueCrawler.action_registry.register('update-crawler')
+class UpdateGlueCrawler(BaseAction):
+    """Updates glue crawlers based on specified parameter
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: glue-crawler-update
+                resource: aws.glue-crawler
+                filters:
+                  - type: value
+                    key: CrawlerSecurityConfiguration
+                    value: null
+                actions:
+                  - type: update-crawler
+                    attributes:
+                        CrawlerSecurityConfiguration: glue-approved
+
+    """
+
+    schema = type_schema(
+        'update-crawler',
+        attributes={'type': 'object'},
+        required=('attributes',))
+
+    permissions = ('glue:UpdateCrawler',)
+    shape = 'UpdateCrawlerRequest'
```

Comment:
please switch from shape to jsonschema, afaics we can omit targets in this context since its app configuration

Reply:
Thanks, i'll get this fixed today